recipes/mariadb-connector-python/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set name = "mariadb" %}
+{% set version = "1.0.7" %}
+
+package:
+  name: {{ name|lower }}
```

Perhaps it'll be less confusing if this package name includes Python:
```suggestion
  name: {{ name|lower }}-python
```